js/components/Todo/TodoInput.js

```diff
@@ -0,0 +1,49 @@
+import Component from "../../core/component.js";
+import store from '../../store/index.js';
+import {keyboardKey} from "../../constants/constants.js";
+import {addTeamToMemberToTodoItem} from "../../service/TodoApi.js";
+
+export default class TodoInput extends Component {
+    constructor(element) {
+        super({
+            store,
+            element
+        });
+    }
+
+    todoInputTemplate = (memberId) => `<input class="new-todo" placeholder="할 일을 입력해주세요." data-member-id="${memberId}" autofocus />`;
+
+
+    render() {
+
+        this.element.querySelectorAll('.input-container').forEach((node) => {
+
+            node.innerHTML = this.todoInputTemplate(node.dataset.memberId);
+
+
+        });
+
+
+        this.element.querySelectorAll('.new-todo').forEach((node) => {
+            node.addEventListener('keyup', async ({key, target}) => {
+                if (key === keyboardKey.Enter) {
+
+                    if (target.value !== '') {
+                        const response = await addTeamToMemberToTodoItem(store.state.selectedTeam._id, node.dataset.memberId ,target.value);
+                        const newTodoItem = {
+                            memberId : node.dataset.memberId,
+                            todoList : response
+                        };
+                        store.dispatch('addMemberTodoItem', newTodoItem);
+                        target.value = '';
+                    }
+
+                    target.value = '';
+                }
```

Comment:
```suggestion
                if (key !== keyboardKey.Enter || target.value === '') return;

              
                        const response = await addTeamToMemberToTodoItem(store.state.selectedTeam._id, node.dataset.memberId ,target.value);
                        const newTodoItem = {
                            memberId : node.dataset.memberId,
                            todoList : response
                        };
                        store.dispatch('addMemberTodoItem', newTodoItem);
   
                    

                    target.value = '';
                
```
이렇게 하면 if 문을 줄일 수 있을 것 같아요